.github/workflows/pr_build_images.yaml

```diff
@@ -14,12 +14,15 @@ jobs:
             test_image: pulp-fedora31
           - test_type: pulp_galaxy_ng
             test_image: pulp-galaxy-ng
+          - test_type: pulp
+            test_image: pulp
```

Comment:
```suggestion
          - test_type: pulp_ci_centos
            test_image: pulp-ci-centos
```

Reply:
I would like us to publish an image called pulp/pulp-ci-centos and pulp/pulp. I also want to continue publishing the pulp/pulp-ci and pulp/pulp-fedora31 images.

Reply:
oh! now I got it! yeah, you are right!